lib/src/color_picker.dart

```diff
@@ -10,17 +10,34 @@ import 'rgb_picker.dart';
 import 'swatches_picker.dart';
 import 'wheel_picker.dart';
 
+enum Picker {
+  swatches,
+  rgb,
+  hsv,
+  wheel,
+  paletteHue,
+  paletteSaturation,
+  paletteValue,
+}
+
 /// Main color picker including all color pickers of this package
 class ColorPicker extends StatefulWidget {
   const ColorPicker({
     required this.onChanged,
     this.color = Colors.blue,
+    this.initialPicker = Picker.paletteHue,
     Key? key,
   }) : super(key: key);
 
   final ValueChanged<Color> onChanged;
   final Color color;
 
+  /// The first picker widget that is shown.
+  ///
+  /// Use [Picker.swatches], [Picker.rgb], [Picker.hsv], [Picker.wheel],
+  /// [Picker.paletteHue], [Picker.paletteSaturation] or [Picker.paletteValue].
```

Comment:
Could we remove this or make it like:
```suggestion
  /// See also:
  ///
  ///  * [Picker] Enumeration of pickers.
```